network-api/networkapi/buyersguide/fields.py

```diff
@@ -2,6 +2,8 @@
 
 
 class ExtendedYesNoField(models.CharField):
+    # This has been replaced with networkapi.wagtailpages.fields.ExtendedYesNoField
+    # TODO: Remove this class when migrations are squashed next.
```

Comment:
Opened a ticket for this here https://github.com/mozilla/foundation.mozilla.org/issues/6437